app/dashboard/tokens.py

```diff
@@ -21,6 +21,8 @@
     {"addr": "0x2941deaad71adb02b944bd38ebce2f1f4c9a62dc", "name": "COLO", "decimals": 18},
     # from https://github.com/etherdelta/etherdelta.github.io/blob/master/config/main.json
     {"addr": "0x0000000000000000000000000000000000000000", "name": "ETH", "decimals": 18},
+    # for backwards compat with old data, also accept 0x0 as ETH token address
+    {"addr": "0x0",                                        "name": "ETH", "decimals": 18},
```

Comment:
also had to add this since the pgdump of tips has `0x0` for the `tokenAddress` column - i guess its either this or someone who can migrate the old data changes those values - let me know what makes sense!

Reply:
makes sense to me